### PR TITLE
docs: revocation_endpoint

### PR DESCRIPTION
**Pull request by livio-a**

### CODE REVIEW

File: docs/docs/apis/openidoauth/endpoints.md [hunk 1]
```diff
@@ -407,6 +407,79 @@ Check the [Claims](claims) page if a specific claims might be returned and for d
 
 If the token is invalid or expired, an HTTP 401 will be returned.
 
+## revocation_endpoint
+
+[https://api.zitadel.ch/oauth/v2/revoke](https://api.zitadel.ch/oauth/v2/revoke)
+
+This endpoint enables clients to revoke an `acccess_token` or `refresh_token` they have been granted.
```
Comment (fforootd) [inline docs/docs/apis/openidoauth/endpoints.md:414]:
```suggestion
This endpoint enables clients to revoke an `access_token` or `refresh_token` they have been granted.
```

File: docs/docs/apis/openidoauth/endpoints.md [hunk 2]
```diff
@@ -407,6 +407,79 @@ Check the [Claims](claims) page if a specific claims might be returned and for d
...
+
+:::important
+If you revoke an `access_token` only that specific token will be revoked. If you revoke a `refresh_token`,
```
Comment (fforootd) [inline docs/docs/apis/openidoauth/endpoints.md:417]:
```suggestion
If you revoke an `access_token` only the specific token will be revoked. When revoking a `refresh_token`,
```

File: docs/docs/apis/openidoauth/endpoints.md [hunk 3]
```diff
@@ -407,6 +407,79 @@ Check the [Claims](claims) page if a specific claims might be returned and for d
...
+    values={[
+        {label: 'client_secret_basic', value: 'client_secret_basic'},
+        {label: 'client_secret_post', value: 'client_secret_post'},
+        {label: 'none (PKCE)', value: 'none'},
+        {label: 'private_key_jwt', value: 'private_key_jwt'},
+    ]}
+>
+<TabItem value="client_secret_basic">
+
+Send your `client_id` and `client_secret` as Basic Auth Header. Check [Client Secret Basic Auth Method](authn-methods#client-secret-basic) on how to build it correctly.
```
Comment (fforootd) [inline docs/docs/apis/openidoauth/endpoints.md:440]:
```suggestion
Send your `client_id` and `client_secret` as Basic Auth Header. Check [Client Secret Basic Auth Method](authn-methods#client-secret-basic) on how to construct a request correctly.
```

File: docs/docs/apis/openidoauth/endpoints.md [hunk 4]
```diff
@@ -407,6 +407,79 @@ Check the [Claims](claims) page if a specific claims might be returned and for d
...
+Send your `client_id` as parameters in the body:
+
+| Parameter | Description                  |
+| --------- | ---------------------------- |
+| client_id | client_id of the application |
+
+</TabItem>
+<TabItem value="private_key_jwt">
+
+Send a `client_assertion` as JWT for us to validate the signature against the registered public key.
```
Comment (fforootd) [inline docs/docs/apis/openidoauth/endpoints.md:464]:
```suggestion
Send a `client_assertion` as JWT for ZITADEL to verify the signature against the registered public key.
```

File: docs/docs/apis/openidoauth/endpoints.md [hunk 5]
```diff
@@ -407,6 +407,79 @@ Check the [Claims](claims) page if a specific claims might be returned and for d
...
+
+| Parameter             | Description                                                                                                 |
+| --------------------- | ----------------------------------------------------------------------------------------------------------- |
+| client_assertion      | JWT built and signed according to [Using JWTs for Client Authentication](authn-methods#client-secret-basic) |
```
Comment (fforootd) [inline docs/docs/apis/openidoauth/endpoints.md:468]:
```suggestion
| client_assertion      | JWT created and signed according to [Using JWTs for Client Authentication](authn-methods#client-secret-basic) |
```

**Review by fforootd — CHANGES_REQUESTED** [review]
Some small changes

File: docs/docs/apis/openidoauth/endpoints.md (hunk 2 above)
Comment (livio-a) [reply docs/docs/apis/openidoauth/endpoints.md:417]: Is it intentional that one sentence begins with "if" and the other with "when"?

Comment (fforootd) [reply docs/docs/apis/openidoauth/endpoints.md:417]: I tried to replace the twice `If` but it sounds bad. So feel free to ignore this one 😄

Comment (livio-a) [reply docs/docs/apis/openidoauth/endpoints.md:417]: no that's good then, sounds better. just was not sure if it was intended

**Review by fforootd — APPROVED** [review]

**Pull request merged**